src/OpenStudy/study_openinterface.jl

```diff
@@ -195,6 +209,9 @@ function initialize_study(
         log_file = file,
         verbose = verbose,
     )
+    if add_transformers_to_series
+        _merge_collections!(data)
+    end
```

Comment:
another possibility is to simple change the name of this function to something like `_merge_psr_transformer_and_psr_serie`